index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <a href="#">My Logo</a>
+        
+        <nav>
+            <img src="images/Union.png" alt="menu">
+        </nav>
+    </header>
+
+    <section class="body">
+        <h1>I'm Mariane <br> Glad to see  you!</h1>
+
+        <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+
+        <h2>LET'S CONNECT</h2>
+
+        <ul class="social">
+            <li>
+                <a href="#"><img src="images/Linkedin icon.png" alt="icons"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/Vector (1).png" alt="icons"></a>
+            </li>            <li>
+                <a href="#"><img src="images/Vector (2).png" alt="icons"></a>
+            </li>            <li>
+                <a href="#"><img src="images/Vector (3).png" alt="icons"></a>
+            </li>            <li>
+                <a href="#"><img src="images/Vector.png" alt="icons"></a>
+            </li>
+        </ul>
```

Comment:
- [ ] Please rename your files, you can't use name files with spaces, this is making your linters fail. You could rename it as "Vector-(3).png"